source/manifest.json

```diff
@@ -15,7 +15,15 @@
 		"128": "icon.png"
 	},
 	"permissions": [
-		"storage"
+		"storage",
+		"<all_urls>"
+	],
+	"content_scripts": [ {
+			"matches": [ "*://*/*" ],
+			"js":      [ "content.js" ],
+			"css":     ["content.css"],
+			"run_at":  "document_end"
+		}
```

Comment:
Weird indentation here for the opening and closing braces, please fix.

Reply:
Spacing and indentation hasn't been fixed here